goodtablesio/integrations/github/blueprint.py

```diff
@@ -89,7 +100,11 @@ def github_settings():
                               GithubRepo.name).
                      all())
 
-    return render_template('github_settings.html', sync=sync, repos=repos)
+    return render_template('index.html', route='GithubSettings', props={
+        'userName': getattr(current_user, 'display_name', None),
+        'sync': sync,
+        'repos': repos,
```

Comment:
This is not JSON serializable so I'm getting an exception on `/github/settings`. Maybe use `[r.to_json() for r in repos]`?